src/delta_lake/spark_utils.py

```diff
@@ -0,0 +1,213 @@
+"""
+Spark utilities for the Delta Lake service.
+"""
+
+## Copied from https://github.com/kbase/cdm-jupyterhub/blob/main/src/spark/utils.py
+
+import os
+import socket
+from datetime import datetime
+from typing import Dict, List, Optional
+from urllib.parse import urlparse
+
+from pyspark.conf import SparkConf
+from pyspark.sql import SparkSession
+
+from src.service.arg_checkers import not_falsy
+
+# the default number of CPU cores that each Spark executor will use
+# If not specified, Spark will typically use all available cores on the worker nodes
+DEFAULT_EXECUTOR_CORES = 1
+# Available Spark fair scheduler pools are defined in /config/spark-fairscheduler.xml
+SPARK_DEFAULT_POOL = "default"
+SPARK_POOLS = [SPARK_DEFAULT_POOL, "highPriority"]
+DEFAULT_MAX_EXECUTORS = 5
+
+
+def _get_jars(jar_names: List[str]) -> str:
+    """
+    Helper function to get the required JAR files as a comma-separated string.
+
+    :param jar_names: List of JAR file names
+
+    :return: A comma-separated string of JAR file paths
+    """
+    jar_dir = not_falsy(os.getenv("SPARK_JARS_DIR"), "SPARK_JARS_DIR")
```

Comment:
in the original method, it was set explicitly
```
JAR_DIR = '/opt/bitnami/spark/jars'
```
https://github.com/kbase/cdm-jupyterhub/blob/main/src/spark/utils.py#L14

Reply:
is this already set in the dockerfile then I assume?